features/step_definitions/ssp_steps.rb

```diff
@@ -75,11 +106,19 @@
 end
 
 Then /^I should be on the g7 supplier dashboard page$/ do
```

Comment:
Should this be changed to something like `I should be on the framework dashboard page`, as it is no longer G7-specific.